examples/plugins/ebpf/http-requests.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+import bcc
+
+import time
+import datetime
+import os
+import signal
+import errno
+import json
+import urlparse
+import threading
+import socket
+import BaseHTTPServer
+import SocketServer
+import sys
+
+EBPF_FILE = "http-requests.c"
+EBPF_TABLE_NAME = "received_http_requests"
+PLUGIN_UNIX_SOCK = "/var/run/scope/plugins/http_requests.sock"
+
+class KernelInspector(threading.Thread):
+    def __init__(self):
+        super(KernelInspector, self).__init__()
+        self.bpf = bcc.BPF(EBPF_FILE)
+        self.http_rate_per_pid = dict()
+        self.lock = threading.Lock()
+
+    def update_http_rate_per_pid(self, last_req_count_snapshot):
+        new_req_count_snapshot = dict()
+        new_http_rate_per_pid = dict()
+        req_count_table = self.bpf.get_table(EBPF_TABLE_NAME)
+        for key, value in req_count_table.iteritems():
+            request_delta = value.value
+            if key.pid in last_req_count_snapshot:
+                 request_delta -= last_req_count_snapshot[key.pid]
+            if request_delta > 0:
+                new_http_rate_per_pid[key.pid] = request_delta
```

Comment:
(we could smooth this out with an averaging sliding window of a few seconds though).